### PR TITLE
ADMINAPI-103 | List of applications for selected Ods Instance

### PR DESCRIPTION
**Pull request by lchavarriaga**

### CODE REVIEW

File: Application/EdFi.Ods.AdminApi/Features/Applications/ReadApplicationsByOdsInstance.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    {
+        var url = "odsinstances/{id}/applications";
+
+        AdminApiEndpointBuilder.MapGet(endpoints, url, GetOdsInstanceApplications)
+            .WithDescription("Retrieves applications assigned to a specific ODS instance based on the resource identifier.")
+            .WithRouteOptions(b => b.WithResponse<ApplicationModel[]>(200))
+            .BuildForVersions(AdminApiVersions.V1);
+    }
+
+    internal Task<IResult> GetOdsInstanceApplications(GetApplicationsByOdsInstanceIdQuery getApplicationByOdsInstanceIdQuery, IMapper mapper, int id)
```
Comment (jagudelo-gap) [inline Application/EdFi.Ods.AdminApi/Features/Applications/ReadApplicationsByOdsInstance.cs:24]: The parameter _getApplicationByOdsInstanceIdQuery_ should be the interface and not the interface implementation

**Review by jagudelo-gap — APPROVED** [review]
Approved with a suggestion for file ReadApplicationsByOdsInstance.cs

File: Application/EdFi.Ods.AdminApi/Features/Applications/AddApplication.cs [hunk 2]
```diff
@@ -6,12 +6,12 @@
...
 using EdFi.Ods.AdminApi.Infrastructure;
-using EdFi.Ods.AdminApi.Infrastructure.Documentation;
+using EdFi.Ods.AdminApi.Infrastructure.Commands;
 using EdFi.Ods.AdminApi.Infrastructure.Database.Commands;
+using EdFi.Ods.AdminApi.Infrastructure.Documentation;
 using FluentValidation;
-using Swashbuckle.AspNetCore.Annotations;
 using FluentValidation.Results;
-using EdFi.Ods.AdminApi.Infrastructure.Commands;
+using Swashbuckle.AspNetCore.Annotations;
```
Comment (CSR2017) [inline Application/EdFi.Ods.AdminApi/Features/Applications/AddApplication.cs:14]: @lchavarriaga 
Please revert back these line number changes. Thanks

File: Application/EdFi.Ods.AdminApi/Features/Applications/EditApplication.cs [hunk 3]
```diff
@@ -6,12 +6,12 @@
 using AutoMapper;
 using EdFi.Admin.DataAccess.Contexts;
 using EdFi.Ods.AdminApi.Infrastructure;
-using EdFi.Ods.AdminApi.Infrastructure.Documentation;
+using EdFi.Ods.AdminApi.Infrastructure.Commands;
 using EdFi.Ods.AdminApi.Infrastructure.Database.Commands;
+using EdFi.Ods.AdminApi.Infrastructure.Documentation;
 using FluentValidation;
 using FluentValidation.Results;
 using Swashbuckle.AspNetCore.Annotations;
```
Comment (CSR2017) [inline Application/EdFi.Ods.AdminApi/Features/Applications/EditApplication.cs:14]: @lchavarriaga 
same here. Thanks

File: Application/EdFi.Ods.AdminApi/Infrastructure/Database/Queries/GetApplicationsByOdsInstanceIdQuery.cs [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using EdFi.Admin.DataAccess.Contexts;
+using EdFi.Admin.DataAccess.Models;
+
+namespace EdFi.Ods.AdminApi.Infrastructure.Database.Queries;
+
+public class GetApplicationsByOdsInstanceIdQuery
+{
```
Comment (CSR2017) [inline Application/EdFi.Ods.AdminApi/Infrastructure/Database/Queries/GetApplicationsByOdsInstanceIdQuery.cs:12]: @lchavarriaga 
Please add IGetApplicationsByOdsInstanceIdQuery interface. Thanks

**Review by CSR2017 — APPROVED** [review]

**Pull request merged**